tfe/logging.go

```diff
@@ -33,10 +35,21 @@ func IsDebugOrHigher() bool {
 // RoundTrip is a transport method that logs the request and response if the TF_LOG level is
 // TRACE or DEBUG
 func (t *loggingTransport) RoundTrip(req *http.Request) (*http.Response, error) {
+	includeBody := true
+
+	bodyHasSensitiveVals, err := findSensitiveValues(req)
+	if err != nil {
+		return &http.Response{}, err
+	}
+
+	if bodyHasSensitiveVals {
+		includeBody = false
+	}
```

Comment:
OK this is just an optional simplification I have in mind but what if you ignored errors from findSensitiveValues and returned false in those circumstances? Then all this could become:

```suggestion
	includeBody = !hasSensitiveValues(req)
```

Much more readable and very low risk since the regex pattern is literal. You could add a panic for not being able to read the request body.

Reply:
I like this suggestion!

Reply:
Sorry, you are correct. That part is not returning an error and already panics if it doesn't compile.